ovn_cluster.sh

```diff
@@ -514,7 +547,12 @@ function start() {
             start-db-cluster
             central=${CENTRAL_NAME}-1
         else
-            ${RUNC_CMD} exec ${CENTRAL_NAME} ${OVNCTL_PATH} start_northd
+            if check_etcd_mode; then
+                start_etcd
+                ${RUNC_CMD} exec ${CENTRAL_NAME} ${OVNCTL_PATH} --ovn-manage-ovsdb=no start_northd
```

Comment:
Special attention will also be needed when stopping northd, right?

Reply:
> Special attention will also be needed when stopping northd, right?

correct but I am not there yet, just trying to get it working